src/components/token/TokenAlerts.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+
+
+const TokenAlerts = (props) => {
+  const [token, setToken] = useState(props.token);
+
+  useEffect(() => {
+    setToken({banned: true});
```

Comment:
Testing code? I feel it will get the flags from the `props`, right?

Reply:
changed